queries/typescript/textobjects.scm

```diff
@@ -1 +1,5 @@
 ; inherits: javascript
+
+;; parameters
+(formal_parameters
```

Comment:
Is there also optional parameters when there is  required_parameter?

Reply:
```
    "children": {
      "multiple": true,
      "required": false,
      "types": [
        {
          "type": "optional_parameter",
          "named": true
        },
        {
          "type": "required_parameter",
          "named": true
        },
        {
          "type": "rest_parameter",
          "named": true
        }
```

Reply:
Maybe just use any child `(_)`?

Reply:
Yeah I thought so too, but in my examples I always got required_parameter. For example, this gives me two `required_parameter`s

```typescript
  doThing(p1: number, p2: number | undefined = undefined): number {
    return p2 ?? p1
  }
```

I wonder if it's an issue in the parser?